src/main/java/com/rybka/view/ExchangeView.java

```diff
@@ -41,9 +50,24 @@ public void showDialog() {
                     total);
 
             hibernateDAO.save(convertedResult);
-            hibernateDAO.showTableRow();
+            var history = hibernateDAO.select();
 
             showExchange(convertedResult);
+
+            switch (userExportChoice) {
```

Comment:
Let's look in the task description 
`According to documentation user should be able to export history of his exchanges that recorded to the DB.
Implement services that can export data to the CSV, JSON, Console.`

Why do you add this code?